addons/sale/tests/test_payment_automatic_invoice.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+from freezegun import freeze_time
+
+from .test_sale_common import TestCommonSaleNoChart
+
+class PaymentTest(TestCommonSaleNoChart):
+    @classmethod
+    def setUpClass(cls):
+        super(PaymentTest, cls).setUpClass()
+        cls.setUpClassicProducts()
+
+        cls.env['ir.config_parameter'].sudo().set_param('sale.automatic_invoice', True)
+
+        journal = cls.env['account.journal'].create({'type': 'bank', 'name': 'bank', 'code': 'BANK'})
+        cls.payment_test = cls.env.ref('payment_test.payment_acquirer_test').create({
+            'name': "test",
+            'journal_id': journal.id,
+        })
```

Comment:
Why a new test class ?
Why a class setup if there is only one test ?
Why is the class named 'PaymentTest' ?